.github/workflows/examples-tizen.yaml

```
@@ -61,3 +67,20 @@ jobs:
                           build \
                           --copy-artifacts-to out/artifacts \
                       "
+
+            - name: Bloat report - chip-tool
+              run: |
+                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
+                    linux arm chip-tool-ubsan out/tizen-arm-chip-tool-ubsan/chip-tool \
```

should we have tizen somewhere in the platforms? Linux seems to clash with the "ubuntu" builds of things.

I believe these triplets are used to uniquely identify things for reports. 

```suggestion
                    tizen arm chip-tool-ubsan out/tizen-arm-chip-tool-ubsan/chip-tool \
```

---

I think that this is for the configuration which should be selected, but it might be also for some ID... So, maybe you are right that it would be safer to use platform specific name here.

---

The first three arguments correspond to the **platform**, **config**, and **target** headings in the comment tables. The latter two are used to match up corresponding runs, but the first (platform) _also_ identifies a config file under `scripts/tools/memory/platform`. You can probably just copy `linux.cfg` to `tizen.cfg`. For the GitHub bloat reports now, the important part is defining the sections that go into the FLASH and RAM totals.